### PR TITLE
[JFG-944] first fast smoke

### PR DESCRIPTION
**Pull request by oskliarov**

### CODE REVIEW

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaasSmokeTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    public JLoadScenario getHttpScenarios(){
+
+        JLoadTest getSessionsTest = getTest("get sessions", new JHttpQuery().get().path("/jaas/db/sessions"));
+        JLoadTest getProjectsTest = getTest("get projects", new JHttpQuery().get().path("/jaas/projects"));
+        JLoadTest postProjectTest = getTest("POST", new JHttpQuery<String>()
+                .post().body("{\"dbId\": 1,\"description\": \"descr\",\"version\": \"2\",\"zipPath\":\"\"}")
+                .header("Content-Type", "application/json")
+                .path("/jaas/projects"));
+
+// TODO rework with custom invoker
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaasSmokeTest.java]: add JFG ticket number to TODO

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+//  .addListeners(new NotNullInvocationListener()) TODO JFG-979
+//  .addMetrics("metric-success-rate", "metric-not-null-response")
+//  .withQueryDistributor("query-distributor-round-robin") TODO JFG-962
+                .build();
+
+
+        JLoadProfile load = JLoadProfileRps.builder(RequestsPerSecond.of(100)).withMaxLoadThreads(1).withWarmUpTimeInSeconds(1).build();
+
+        JTerminationCriteria termination = new JTerminationCriteriaIterations(IterationsNumber.of(1000), MaxDurationInSeconds.of(500));
+
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java]: why so long 500 seconds?

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    private Iterable<JHttpEndpoint> getEndpoints() {
+        // TODO oskliarov: when JFG-972 will be done use properties
+        return Collections.singletonList(new JHttpEndpoint("http://localhost:8080"));
+    }
+
+    private Iterable<JHttpQuery> getQueries() {
+        // TODO oskliarov: when JFG-972 will be done use properties
+        return Stream.of("55", "12", "77").map(q -> new JHttpQuery().get().path("/sleep", q)).collect(Collectors.toList());
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java]: please add comment what is the meaning of these queries

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                .withValidators(Collections.singletonList(NotNullResponseValidator.class))
+//   TODO add the following things after clarifying
+//  .addListeners(new NotNullInvocationListener()) TODO JFG-979
+//  .addMetrics("metric-success-rate", "metric-not-null-response")
+//  .withQueryDistributor("query-distributor-round-robin") TODO JFG-962
+                .build();
+
+
+        JLoadProfile load = JLoadProfileRps.builder(RequestsPerSecond.of(100)).withMaxLoadThreads(1).withWarmUpTimeInSeconds(1).build();
+
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java]: max thread = 1. will it be enough?

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaasSmokeTest.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.MaxDurationInSeconds;
+
+import java.util.Collections;
+
+
+public class JaasSmokeTest {
+    public JLoadScenario getHttpScenarios(){
+
+        JLoadTest getSessionsTest = getTest("get sessions", new JHttpQuery().get().path("/jaas/db/sessions"));
+        JLoadTest getProjectsTest = getTest("get projects", new JHttpQuery().get().path("/jaas/projects"));
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaasSmokeTest.java]: we will remove /projects from API

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/config/JLoadScenariosConfig.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+public class JLoadScenariosConfig {
+    
+    @Bean
+    public JLoadScenario jaasSmokeTest() {
+        return new JaasSmokeTest().getHttpScenarios();
+    }
+
+    @Bean
+    public JLoadScenario jaagerSmokeTest() {
+        return new JaggerSmokeTest().jTestSuites();
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/config/JLoadScenariosConfig.java]: please, use same naming for methods. now you have test suits and http scenarios

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/config/JLoadScenariosConfig.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration
+public class JLoadScenariosConfig {
+    
+    @Bean
+    public JLoadScenario jaasSmokeTest() {
+        return new JaasSmokeTest().getHttpScenarios();
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/config/JLoadScenariosConfig.java]: getHttpScenarios
why plural? you are returning single scenario

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaasSmokeTest.java [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+import java.util.Collections;
+
+
+public class JaasSmokeTest {
+    public JLoadScenario getHttpScenario(){
+
+        JLoadTest getSessionsTest = getTest("get sessions", new JHttpQuery().get().path("/jaas/db/sessions"));
+        JLoadTest getExecutionsTest = getTest("get execution", new JHttpQuery().get().path("/jaas/executions"));
+        JLoadTest postProjectTest = getTest("POST", new JHttpQuery<String>()
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaasSmokeTest.java]: postProjectTest?

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.MaxDurationInSeconds;
+
+import java.util.Collections;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class JaggerSmokeTest {
+    public JLoadScenario getJaggerScenario() {
+        JTestDefinition sleep_service_5ms_simple_http_query = JTestDefinition
+                .builder(Id.of("sleep-service-5ms-simple-http-query-provider"), getEndpoints())
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java]: why do we need query-provider in the id?

Comment (oskliarov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTest.java]: Why not?

**Pull request merged**